### ISSUE TITLE
Kafka Tenant Level Events require a Webhook URL to be setup

### ISSUE THREAD

**Issue by TheMiniDriver** [FusionAuth/fusionauth-issues#1981]
### Description
When adding a Kafka integration, audit level events are sent through. However, to receive tenant level events (eg. type:`user.create` etc), a webhook must be configured. The events configured for the Webhook are then sent to Kafka. 

Therefore, Kafka events cannot be configured separately from Webhooks.

Furthermore, the webhook endpoint has to actually exist and respond with a 200 (you can't use `https://example.com`) therefore you may be sharing user data in ways you don't want to.

### Community guidelines
All issues filed in this repository must abide by the [FusionAuth community guidelines](https://fusionauth.io/community/forum/topic/1000/code-of-conduct).

**Comment by mooreds** [comment]
This feels like a bug to me, as I'd expect webhooks and kafka to receive events independently.

**Comment by robotdan** [comment]
I agree, you should not need to create a regular webhook to get Kafka to work for tenant scoped events. 

It looks like this should work though - we'll need to try and recreate internally. 
The only time we short circuit and skip trying to send is when we have a tenant scoped event, and the event has not been enabled.

But maybe we need to dig in further to see what is going on.

**Comment by wied03** [comment]
In our testing, this isn't a problem (not sure what version this was reported on), so we are closing this issue. @TheMiniDriver if you have any further version details to share though about when this issue was noticed, we would love to hear about them. Thanks.
